jmix-dynattr/dynattr-flowui/src/main/java/io/jmix/dynattrflowui/view/categoryattr/CategoryAttributesDetailView.java

```diff
@@ -312,6 +312,34 @@ protected void onAfterShow(BeforeShowEvent event) {
         loadTargetViews();
     }
 
+    @Install(to = "nameField", subject = "validator")
+    protected void nameFieldUniqueNameValidator(String value) {
+        validateUniqueStringOnAttribute(value, CategoryAttribute::getName, "notUniqueAttributeName");
```

Comment:
Could you please check using the debugger how much time this validator is fired on each textField value change? I have a suspicion that it is invoked more than once.

Reply:
3 times invoking name validator (if code wasnt empty)
4 times invoking name validator + 1 time code validator (if code was empty)

Reply:
UPD message
![image](https://github.com/jmix-framework/jmix/assets/43281522/3b690eeb-bfa3-4857-8dae-27bffb0c6349)

Reply:
> 4 times invoking name validator

Do you have an idea why this happens? Shouldn't it be called just once?

Reply:
> > 4 times invoking name validator
> 
> Do you have an idea why this happens? Shouldn't it be called just once?

Here code `CategoryAttributesDetailView::refreshAttributesValues` that refrashing and triggering recalculations and because if it we have `isRefreshing` variable.

But i think its ok that validator invoking more than one time

Reply:
@gorbunkov All fine, invoking only one time